### PR TITLE
Test command

### PR DESCRIPTION
**Pull request by admindgard**

### CODE REVIEW

File: tests/unit/test_api_service.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# import requests
+from requests.exceptions import HTTPError
+import requests_mock
+import pytest
+from ...src.mindgard.constants import API_BASE
+from ...src.mindgard.api_service import ApiService
+
+def test_submit_test(requests_mock: requests_mock.Mocker) -> None:
+    svc = ApiService()
```
Comment (danny-hunt) [inline tests/unit/test_api_service.py:9]: svc = service(?)

File: tests/unit/test_api_service.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    requests_mock.post(f"{API_BASE}/assessments", text='{}', status_code=201)
+    svc.submit_test("testtoken", "cfp_faces")
+
+def test_submit_test_raises_failure(requests_mock: requests_mock.Mocker) -> None:
+    svc = ApiService()
+    requests_mock.post(f"{API_BASE}/assessments", text='{}', status_code=400)
```
Comment (danny-hunt) [inline tests/unit/test_api_service.py:15]: This looks cool!

File: tests/unit/test_test_command.py
Comment (danny-hunt) [inline tests/unit/test_test_command.py:1]: Love it

File: src/mindgard/__main__.py [hunk 3]
```diff
@@ -75,9 +76,9 @@ def main() -> None:
...
         exit(res.code())
     elif args.command == 'test':
-        if args.target is None:
-            raise Exception("test command requires target argument")
-        res = run_test(target_name=args.target, json_format=bool(args.json), risk_threshold=int(args.risk_threshold))
+        api_service = ApiService()
+        cmd = RunTestCommand(api_service)
+        res = cmd.run(model_name=args.target, json_format=bool(args.json), risk_threshold=int(args.risk_threshold))
         exit(res.code())
     elif args.command == 'tests':
```
Comment (danny-hunt) [inline src/mindgard/__main__.py:83]: I'd be down to kill the old stuff asap

**Review by danny-hunt — APPROVED** [review]

File: src/mindgard/__main__.py (hunk 3 above)
Comment (admindgard) [reply src/mindgard/__main__.py:83]: yep lets tidy next.

**Pull request merged**